src/deps/uws/BodyReaderMixin.zig

```diff
@@ -41,32 +41,47 @@ pub fn BodyReaderMixin(
 
         fn onData(ctx: *@This(), resp: uws.AnyResponse, chunk: []const u8, last: bool) !void {
             if (last) {
-                var body = ctx.body; // stack copy so onBody can free everything
-                resp.clearAborted();
+                // Free everything after
+                var body = ctx.body;
```

Comment:
this copy of `body` leaks because `ctx.body` is immediately reassigned and `onBody` does not free the chunk. maybe `defer body.deinit()`?